src/plugin/VideoRoomPublisherJanusPlugin.js

```diff
@@ -285,6 +285,16 @@ class VideoRoomPublisherJanusPlugin extends JanusPlugin {
       return
     }
 
+    if (videoroom === 'talking') {
+      this.emit('talking')
+      return
+    }
+
+    if (videoroom === 'stopped-talking') {
+      this.emit('stopped-talking')
```

Comment:
Ez miért nem lett camelCase? Úgy látom a többi event is olyan.

Reply:
Mert így jön a janusból. De átírom teveÜgyre.